compiler/src/dmd/declaration.h

```diff
@@ -600,6 +600,7 @@ class FuncDeclaration : public Declaration
     // set if someone took the address of this function
     int tookAddressOf;
     bool requiresClosure;               // this function needs a closure
+    bool skipCodegen;                   // do not generate code for this function
```

Comment:
This is redundant with `FUNCFLAG.isCompileTimeOnly`

Reply:
Unfortunately, isCompileTimeOnly doesn't do what it says it does. That logic is a mess. So I avoided it pending a future refactoring.